app/views/comfy/admin/api_namespaces/show.html.haml

```diff
@@ -130,6 +130,14 @@
       = link_to 'Duplicate with associations', duplicate_with_associations_api_namespace_path(id: @api_namespace.id), method: :post, target: '_blank', class: 'btn btn-primary'
     .my-3
       = link_to 'Duplicate without associations', duplicate_without_associations_api_namespace_path(id: @api_namespace.id), method: :post, target: '_blank', class: 'btn btn-primary'
+    .my-3
+      = link_to 'Export JSON with associations', export_with_associations_as_json_api_namespace_path(id: @api_namespace.id), method: :get, class: 'btn btn-primary'
+    .my-3
+      = link_to 'Export JSON without associations', export_without_associations_as_json_api_namespace_path(id: @api_namespace.id), method: :get, class: 'btn btn-primary'
+    .my-3
+      = form_with(url: import_as_json_api_namespaces_path, method: :post, multipart: true) do
```

Comment:
@alis-khadka this import button should be moved to the #new view because you aren't doing anything to the targeted API Namespace when you do an import-- what do you think?
<img width="1728" alt="Screen Shot 2022-06-26 at 8 14 15 AM" src="https://user-images.githubusercontent.com/35935196/175813598-57622a35-7e4e-443a-b6e8-14a65e8553ba.png">

Reply:
@donrestarone 
Yes, I do agree as the import functionality is not related to any specific api_namespace.

I will move the button for importing to **new** view. 

Shall we add this button to **index** view as well?
![image](https://user-images.githubusercontent.com/25191509/175814989-371937c9-ca31-4802-a783-6c2bec3f0c74.png)